py-polars/tests/unit/constructors/test_constructors.py

```diff
@@ -958,11 +958,6 @@ def _constructor(self) -> type:
     assert df.schema == {"colx": pl.Datetime("us")}
     assert df.rows() == [(datetime(2022, 10, 31, 10, 30, 45, 123456),)]
 
-    # pandas is not available
-    monkeypatch.setattr(pl.dataframe.frame, "_check_for_pandas", lambda x: False)
-    with pytest.raises(TypeError):
-        pl.DataFrame(pandas_df)
-
```

Comment:
This no longer raises with recent pandas (v2.2 and later, available on recent python versions) because pandas implements the pycapsule interface, so importing a `pandas.DataFrame` will still work even without using any pandas APIs.

Let me know if you have a preferred way to update this test instead of just removing it.

Reply:
Mayb add a test for what you describe then?